xbmc/addons/AddonManager.cpp

```diff
@@ -111,6 +112,19 @@ bool CAddonMgr::Init()
     return false;
   }
 
+  m_trustedAddonRepos = StringUtils::Split(CCompileInfo::GetTrustedAddonRepos(), ',');
+  if (m_trustedAddonRepos.size() != 0)
```

Comment:
No need for condition here. Just the for loop is ok. I think listing each trusted repo is ok as debug but maybe no value in logging no trusted repos.